sentry/src/main/java/io/sentry/Baggage.java

```diff
@@ -0,0 +1,181 @@
+package io.sentry;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@ApiStatus.Experimental
+public final class Baggage {
```

Comment:
Could you add `final` to the respective method parameters for this class? almost all of the methods are missing it

Reply:
Will do

Reply:
Updated